commands/utils/index.ts

```diff
@@ -9,12 +9,14 @@ export const getEsClient = () => {
     !(config.elastic.username && config.elastic.password)
   )
     return client;
-  const auth = {}
+  let auth;
```

Comment:
Redefining it because the API version and basic auth version are not compatible with each other.